.github/scripts/install_dependencies.sh

```diff
@@ -31,31 +31,27 @@ sudo apt install -y \
   libxft-dev \
   libxml++2.6-dev \
   libreadline-dev \
-  tcl-dev \
+  tcllib \
+  tcl8.6-dev \
   libffi-dev \
   perl \
+  pkg-config \
   texinfo \
   time \
   valgrind \
   zip \
-  qt5-default \
+  qtbase5-dev \
   uuid-dev \
   default-jdk \
-  clang-format-7 \
-  g++-7 \
-  gcc-7 \
-  g++-8 \
-  gcc-8 \
+  clang-format-12 \
```

Comment:
Add clang. Default clang version is v14

Reply:
Now the supported clang-format under Ubuntu 22.04 is at least clang-format-11, while the default is clang-format-14

https://packages.ubuntu.com/search?suite=jammy&section=all&arch=any&keywords=clang-format&searchon=names

While we have hardcoded clang-format-7 in CMakelist:

https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/99d71d90caf14b0711d3f04c25cd665e0abfabfb/cmake/modules/AutoClangFormat.cmake#L28